### PR TITLE
Added paths to log and use spawn

### PR DESCRIPTION
**Pull request by CrashCodes**
Added path information to log results.
Use spawn instead of exec to avoid max buffer issue when making many calls.

### CODE REVIEW

File: src/log.js [hunk 1]
```diff
@@ -1,44 +1,60 @@
...
+
+  var child = spawn('svn', args, {cwd: this.local});
+  var stdout = "";
+
+  child.stdout.on('data', function (data) {
+    stdout += data;
+  });
+
+  child.stderr.on('data', function (data) {
+    console.log('stderr: ' + data);
```
Comment (courajs) [inline src/log.js]: Could you change this message so it's more clear it's coming from the svn package?
Something like `'svn error output:'`

File: src/log.js [hunk 2]
```diff
@@ -1,44 +1,60 @@
...
+  });
+
+  child.on('close', function (code) {
+    if (code !== 0) {
+      console.log('ps process exited with code ' + code);
```
Comment (courajs) [inline src/log.js]: Same as above, can this be something more like `'svn log exited with code '`?

**Comment by courajs** [comment]
Thanks for the PR! I'll merge and release if you update the error messages.
Also, it looks like your editor changed a lot of tabs to spaces. If you could try to prevent that in the future, it will make your PRs easier to review

**Comment by CrashCodes** [comment]
Good suggestions on the error messages. 
Sorry about the whitespace. 
btw, this is my first PR.

**Comment by courajs** [comment]
Awesome, thanks for opening it! Welcome!
This was my first npm module, and first bit of open source too 😄

**Comment by courajs** [comment]
0.3.0 released! I don't really use this package personally anymore, so please let me know if any issues pop up

**Pull request merged**